styles/main_styles.css

```diff
@@ -364,4 +365,127 @@ li {
     top: -5%;
     width: 10.1px;
   }
+
+  /* work section */
+  .works-container {
+    display: grid;
+    grid-template-columns: auto;
+    padding: 70px 24px;
+  }
+
+  .work-item {
+    border-radius: 5px;
+    background-color: white;
+    padding: 10px;
+    margin-top: 70px;
+  }
```

Comment:
- [ ] Following the project requirements,  You should align each project with flexbox.